src/test/java/com/selfxdsd/selfweb/api/output/JsonTaskTestCase.java

```diff
@@ -124,35 +119,11 @@ public void hasEstimation(){
         
     }
     
-    
-    /**
-     * JsonTask hasn't the issueId attribute
-     */
-    
-    @Test
-    public void hasNullIssueId() {
```

Comment:
@Lumi3011 I removed this test since it's the same as "hasIssueId" -- it's still a String :D